compiler/rustc_middle/src/ty/consts/int.rs

```diff
@@ -1,31 +1,33 @@
-use crate::mir::interpret::truncate;
-use rustc_target::abi::Size;
+use crate::mir::interpret::{sign_extend, truncate, InterpResult};
```

Comment:
Is there some better place for `sign_extend` and `truncate` so that we can entirely avoid importing the interpreter here?

Reply:
I made them inherent methods on `Size`... I hope that is in the spirit of what you asked for